KD_Lib/utils/pipeline.py

```diff
@@ -0,0 +1,102 @@
+from itertools import islice
+from tqdm import tqdm
+import time
+
+from KD_Lib.common import BaseClass
+
+
+class Pipeline():
+    """
+    Pipeline of knowledge distillation, pruning and quantization methods
+    supported by KD_Lib. Sequentially applies a list of methods on the student model.
+    
+    All the elements in list must implement either train_student, prune or quantize
+    methods.
+
+    :param: steps (list) list of KD_Lib.KD or KD_Lib.Pruning or KD_Lib.Quantization
+    :param: epochs (int) number of iterations through whole batch for each method in
```

Comment:
We can remove this parameter right? Since each individual method will have epochs in its initialization

Reply:
Distillation specifies these parameters in ```train_student``` method instead of class constructor.